octavia_f5/controller/worker/controller_worker.py

```diff
@@ -612,7 +612,17 @@ def add_loadbalancer(self, load_balancer_id):
 
         selfips = list(chain.from_iterable(
             self.network_driver.ensure_selfips(loadbalancers, CONF.host, cleanup_orphans=False)))
-        self.l2sync.ensure_l2_flow(selfips, network_id)
+
+        # If other LBs in the same network already exist on this host, just ensure correct selfips and subnet routes
+        loadbalancers_here = [lb for lb in loadbalancers
+                              if lb.server_group_id == CONF.host and lb.id != load_balancer_id]
+        LOG.debug(f'LBs already on this device (without {load_balancer_id}): {loadbalancers_here}')
```

Comment:
You don't want to print `loadbalancers_here`, it's the full dictionary.

Reply:
It's not a dict, but a list. But yeah, apparently we have some hosts that have several hundred LBs in one network. Good point.